moveit_py/src/moveit/moveit_core/planning_scene/planning_scene.cpp

```diff
@@ -0,0 +1,405 @@
+               &planning_scene::PlanningScene::knowsFrameTransform, py::const_),
+           py::arg("robot_state"), py::arg("frame_id"),
+           R"(
+           Check if a transform to the frame id is known.
+           This will be known if id is a link name, an attached body id or a collision object.
+           Args:
+               robot_state (:py:class:`moveit_py.core.RobotState`): The robot state to check.
+               frame_id (str): The frame id to check.
+           Returns:
+               bool: True if the transform is known, false otherwise.
+           )")
+
+      .def("knows_frame_transform",
+           py::overload_cast<const std::string&>(&planning_scene::PlanningScene::knowsFrameTransform, py::const_),
+           py::arg("frame_id"),
+           R"(
+           Check if a transform to the frame id is known.
+           This will be known if id is a link name, an attached body id or a collision object.
+           Args:
+               robot_state (:py:class:`moveit_py.core.RobotState`): The robot state to check.
+               frame_id (str): The frame id to check.
+           Returns:
+               bool: True if the transform is known, false otherwise.
+           )")
+
+      .def("get_frame_transform", &moveit_py::bind_planning_scene::get_frame_transform, py::arg("frame_id"),
+           R"(
+           Get the transform corresponding to the frame id.
+           This will be known if id is a link name, an attached body id or a collision object. Return identity when no transform is available.
+           Args:
+               frame_id (str): The frame id to get the transform for.
+           Returns:
+               :py:class:`numpy.ndarray`: The transform corresponding to the frame id.
+           )")
+
+      // writing to the planning scene
+      .def("process_planning_scene_world", &planning_scene::PlanningScene::processPlanningSceneWorldMsg, py::arg("msg"),
+           R"(
+	   Process a planning scene world message.
+	   Args:
+	       msg (:py:class:`moveit_msgs.msg.PlanningSceneWorld`): The planning scene world message.
+	   )")
+
+      //.def("apply_planning_scene_world", &moveit_py::bind_planning_scene::apply_planning_scene_world, py::arg("scene"),
+      //     R"(
+      //     Apply a planning scene world message to the current planning scene.
+      //     Args:
+      //         scene (:py:class:`moveit_msgs.msg.PlanningSceneWorld`): The planning scene world message to apply.
+      // )")
+
+      .def("apply_collision_object", &moveit_py::bind_planning_scene::apply_collision_object,
+           py::arg("collision_object_msg"), py::arg("color_msg") = nullptr,
+           R"(
+           Apply a collision object to the planning scene.
+           Args:
+               object (moveit_msgs.msg.CollisionObject): The collision object to apply to the planning scene.
+           color (moveit_msgs.msg.ObjectColor, optional): The color of the collision object. Defaults to None if not specified.
```

Comment:
Weird indent?

Also seems from this point on in the file, you've stopped using `:py:class:` to wrap object names in the args. Do you mean to do it uniformly throughout?

Reply:
[72a07bb](https://github.com/ros-planning/moveit2/pull/1546/commits/72a07bb23cc341713b0d41988497c4055e250451)

Yes I meant to use it uniformly throughout, further work is required to get the documentation into a good state.